packages/plugin-apollo-client/README.md

```diff
@@ -10,6 +10,39 @@ SnapWP's query system and configuration API are built to support any GraphQL cli
 > 🐉 There be dragons!
 > This project is in **active development** and considered _experimental_. Some features may be incomplete, unstable, or subject to change.
 
+## Installation
+
+To integrate plugin-apollo-client within the SnapWP framework:
+
+```bash
+npm install plugin-apollo-client
+```
+
+## Using `plugin-apollo-client` in `snapwp.config.ts`
+
+To use `plugin-apollo-client` with SnapWP, we will configure the query engine by adding the `ApolloClientEngine` as the engine to be used for GraphQL operations.
+
+```ts
+import type { SnapWPConfig } from '@snapwp/core/config';
+import { ApolloClientEngine } from '@snapwp/plugin-apollo-client';
+
+const config: SnapWPConfig = {
+	query: {
+		engine: ApolloClientEngine,
+	},
+};
+
+export default config;
+```
+
+#### Breakdown of the Setup
```

Comment:
Instead of having separate sections, the entire `## Installation ` section should be numbered steps with e.g

----
1.  Add the package to your project's dependencies:

    ```console
    npm install --save @snapwp/plugin-apollo-client
    ```
    
2. Update your `snapwp.config.ts` to use the Query Engine:

    ```diff
    import type { SnapWPConfig } from '@snapwp/core/config';
    + import { ApolloClientEngine } from '@snapwp/plugin-apollo-client';

    const config: SnapWPConfig = {
    +	query: {
    +		engine: ApolloClientEngine,
    +	},
    };

    export default config;
    ```

Reply:
(PS: do we need/want to add a note in step one about installing `@apollo/client` directly too? (or tanstack/query for that readme)